src/azure-cli/azure/cli/command_modules/acr/_format.py

```diff
@@ -217,6 +221,17 @@ def _taskrun_format_group(item):
     ])
 
 
+def _agentpool_format_group(item):
+    return OrderedDict([
+        ('NAME', _get_value(item, 'name')),
+        ('COUNT', _get_value(item, 'count')),
```

Comment:
@qwordy is there a way I can make the table format to write value zero instead of a leaving it blank? I didn't find anything in the documentation.

Reply:
cc: @yungezz

Reply:
never mind. Fixed